tests/unittest/_torch/test_resource_manager.py

```
@@ -504,15 +509,17 @@ def test_adjust_window_sizes_for_vswa(self):
                     100: [0, 1, 2, 3],
                     134: [4, 5, 6, 7, 8],
                 },
+                [100] * 4 + [134] * 5,
                 134,
                 "less_limited_memory_but_clamped_by_max_tokens"),
         ]
 
-        for memory_bytes, expected_window_sizes, max_tokens, description in test_cases:
+        for memory_bytes, expected_window_sizes, expected_max_attention_window_vec, max_tokens, description in test_cases:
```

Ruff (0.12.2)

Line too long (122 > 120)

(E501)